File: db4oj.optional/core/src/com/db4o/filestats/MiscCollector.java
/* Copyright (C) 2004 - 2010  Versant Inc.  http://www.db4o.com */

/**
 * 
 */
package com.db4o.filestats;

import com.db4o.internal.*;

/**
* @exclude
*/
@decaf.Ignore(decaf.Platform.JDK11)
interface MiscCollector {
	long collectFor(LocalObjectContainer db, int id, SlotMap slotMap);
}

File: db4o.net/Db4objects.Db4o.Optional/Db4objects.Db4o/Filestats/MiscCollector.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

#if !SILVERLIGHT
using Db4objects.Db4o.Filestats;
using Db4objects.Db4o.Internal;

namespace Db4objects.Db4o.Filestats
{
	/// <exclude></exclude>
	internal interface IMiscCollector
	{
		long CollectFor(LocalObjectContainer db, int id, ISlotMap slotMap);
	}
}
#endif // !SILVERLIGHT
